pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -123,47 +135,54 @@ func (r *ReplicaSetReconciler) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	r.log.Debug("Building service")
```

Comment:
maybe we should add a little more context to these debug logs, "Building service: namespace/name"